<launch>
	<node name="spline_optimization_example" pkg="ewok_optimization" type="spline_optimization_example3" output="screen">
		<!--<rosparam param="x_pos">[1.0, 20.0, 1.0, 20.0, 1.0]</rosparam> 
		<rosparam param="y_pos">[1.0, 1.0, 1.0, 1.0, 1.0]</rosparam> 
		<rosparam param="z_pos">[5.0, 5.0, 5.0, 5.0, 5.0]</rosparam>-->

		<!-- <rosparam param="x_pos">[0.0, -9.0,  -11.0, -3.0, -9.0, 15.0, 3.0]</rosparam> 
		<rosparam param="y_pos">[0.0, -13.0, -8.0,  4.0,  18.0,  15.0, 4.0]</rosparam> 
		<rosparam param="z_pos">[5.0,  5.0,   5.0,  5.0,  5.0,   5.0,  5.0]</rosparam> -->

		<!-- DuyNguyen: So diem ben opt phai bang so diem offboard -->
		<!-- <rosparam param="x_pos">[0.0, -8.0,  -11.0, -3.0, -10.0]</rosparam> 
		<rosparam param="y_pos">[0.0, -11.0, -8.0,  4.0,  15.0]</rosparam> 
		<rosparam param="z_pos">[5.0,  8.0,  5.0,   8.0,  5.0]</rosparam> -->

		<rosparam param="x_pos">[2.0, 16.0]</rosparam> 
		<rosparam param="y_pos">[2.0, 2.0]</rosparam> 
		<rosparam param="z_pos">[3.0, 3.0]</rosparam>

		<param name="number_of_target" type="int" value="2"/>
		<param name="target_error" type="double" value="0.1"/>
		<param name="odom_error" type="bool" value="true"/> <!--false-->
		<param name="number_of_points" type="int" value="10"/>
		<param name="dt_value" type="double" value="0.4"/>
	</node>
    <!-- <node pkg="tf" type="static_transform_publisher" name="tf_camera" args="0.1 0 0 -1.57 0 -1.57 base_link camera_link 33"/>
	<node pkg="tf" type="static_transform_publisher" name="world_map_linker" args="0 0 0 0 0 0 world map 100" /> -->
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find ewok_optimization)/rviz/optimization_example.rviz" />

</launch>
